src/getModule.js

```diff
@@ -1,5 +1,5 @@
 const Module = typeof __webpack_require__ === 'function'
-  ? require('../webpack/module')
+  ? require('../webpack/src/module')
```

Comment:
any reason to move this file?

Reply:
for the eslint rule, it should apply the same rule with interceptor.js